plugins/org.jboss.tools.vpe.browsersim/src/org/jboss/tools/vpe/browsersim/ui/BrowserSim.java

```diff
@@ -566,6 +570,7 @@ public void reinitSkin() {
 		skin.getBrowser().setUserAgent(device.getUserAgent());
 		skin.getBrowser().setUrl(oldSkinUrl); 
 		skin.getShell().open();
+		processLiveReload(specificPreferences.isEnableLiveReload());

```

Comment:
Basically, this is the fix for livereload. it enables it, if needed, if the 'Refresh' button from the 'Platforms' toolbar was pressed.